scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixEvaluation.java

```diff
@@ -4,19 +4,25 @@
 
 public interface ScalafixEvaluation {
     /**
-     * @return boolean true if scalafix has successfully been evaluated on all files configured and no ScalafixError has been found
+     * @return boolean true if scalafix has successfully been evaluated on all files configured with no error.
      */
     boolean isSuccessful();
 
     ScalafixError[] getErrors();
 
+    /**
+     *
+     * @return a more detailed error message that can be
+     *  - an explanation of why scalafix failed running at all
+     *  - a concatenation of all {@Link ScalafixFileEvaluation} message Errors
```

Comment:
Cosmetic, but I think you need tags for this list of bullet points to appear properly in Javadoc like https://javadoc.io/doc/ch.epfl.scala/scalafix-interfaces/0.9.21/scalafix/interfaces/ScalafixFileEvaluation.html. Have a look at the rest of the Javadoc, it's very clean, so I think we should try to keep the same standard.